### PR TITLE
Scale Integration & Design Update to comply with Brand-Design

### PR DESCRIPTION
**Pull request by SebastianGode**

### CODE REVIEW

File: otcdocstheme/theme/otcdocs/footer.html [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
-      </div>
-    </div>
-  </div>
-</footer>
+</scale-app-shell>
+<scale-app-footer id="footer-shell" variant="standard" claim-lang="en">
+</scale-app-footer>
+<script>
+  const footer_shell = document.getElementById("footer-shell");
+  // footer_shell.footerNavigation = [
```
Comment (tischrei) [inline otcdocstheme/theme/otcdocs/footer.html:6]: delete comments

**Review by tischrei — CHANGES_REQUESTED** [review]

**Comment by SebastianGode** [comment]
Current look if we would merge this PR:
image

**Review by gtema — CHANGES_REQUESTED** [review]
uhhhh, adding 2.1 thousand files is a very very bad way. Can we update build process to get everything what's required without putting those under git?

**Comment by tischrei** [comment]
> uhhhh, adding 2.1 thousand files is a very very bad way. Can we update build process to get everything what's required without putting those under git?

Let's use the CDN for scale which is there since last week...

**Comment by SebastianGode** [comment]
@gtema Removed all the node module stuff and added CDN

**Review by tischrei — DISMISSED** [review]

**Review by tischrei — DISMISSED** [review]

**Review by tischrei — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Pull request merged**